schema/rdl_schema_0.1.json

```diff
@@ -115,31 +127,38 @@
 							],
 							"properties": {
 								"title": {
+									"title": "Resource title",
 									"type": "string",
 									"description": "A title for this dataset."
 								},
 								"description": {
+									"title": "Resource description",
 									"type": "string",
 									"description": "Additional description of the dataset version not already covered by the other resource fields."
 								},
 								"format": {
+									"title": "Format",
```

Comment:
We should consider providing a codelist for this field to make it easier for users and applications to filter by format. One option is to use the [IANA Media Types codelist](https://www.iana.org/assignments/media-types/media-types.xhtml), but we could also look at the approach used in CKAN etc.

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/54#issuecomment-1553966659